README.txt

```diff
@@ -22,3 +22,214 @@ Gallerya transforms the WordPress native post gallery into a full fledged slides
 
 * PHP 7.0 or later.
 
+== Changelog ==
+
+= 1.9.16 =
+2018-10-16
+
+* Fixed images are not visible on mobile.
+
+= 1.9.15 =
+2018-10-05
+
+* Added basic readme file.
```

Comment:
Only official releases should be listed in the changelog.  If I remember correctly, the wp-release script generates the proper release notes to add to the changelog.

See https://github.com/netzstrategen/wordpress-gallerya/releases

This means that all of the entries below should be aggregated into 1.9.15.

And all entries above until 1.10.1 should be aggregated into the release for 1.10.1.

All entries belonging to commits later than 1.10.1 should be removed, as there is no release for them yet, and the new version number depends on the changes that will be committed into it.